src/features/settings/Settings.tsx

```diff
@@ -70,6 +70,10 @@ const Settings = () => {
   const copyText = useCopyText()
   const { showOKAlert, showOKCancelAlert } = useAlert()
   const { data: status } = useGetSolanaStatusQuery()
+  // Override status to always show migration
```

Comment:
Will we need to bring this back eventually then?

Reply:
I left that comment there in case you wanna test if the migration is complete. You can also just run the sentinel locally